vaadin-upload-flow-parent/vaadin-upload-flow-integration-tests/src/test/java/com/vaadin/flow/component/upload/tests/UploadI18nIT.java

```diff
@@ -69,6 +69,7 @@ public void testFullI18nShouldOverrideCompleteConfigurationInWebComponentPropert
 
         UploadI18N expected = UploadTestsI18N.RUSSIAN_FULL;
         JsonObject expectedJson = (JsonObject) JsonSerializer.toJson(expected);
+        deeplyRemoveNullValuesFromJsonObject(expectedJson);
```

Comment:
Please add the new property to `UploadTestsI18N.RUSSIAN_FULL` because otherwise it's not covered by these tests.

Reply:
Added the property to full translation. Updated the partial i18n test to not expect `sizeBase` to be present in the resulting translation map.